src/nos/core/notifications.py

```diff
@@ -126,3 +133,58 @@ def formatted(self, **kwargs) -> "Notification":
     "File upload completed",
     "The file {file_id} has been successfully uploaded.",
 )
+
+ALL_IVAS_INVALIDATED_TO_USER = Notification(
+    "IVA Invalidation",
+    """
+All of your IVAs have been successfully invalidated.
+
+If you have any questions, please contact a Data Steward at GHGA: {email}.
+""",
+)
+
+IVA_CODE_REQUESTED_TO_USER = Notification(
+    "IVA Request Received",
+    """
+Your request for an IVA verification code has been received.
```

Comment:
something like "A verification code will be sent to you soon via the specified contact address."